lib/tableau/router.ex

```diff
@@ -14,7 +16,7 @@ defmodule Tableau.Router do
   plug :rerender
 
   plug Tableau.IndexHtml
-  plug Plug.Static, at: "/", from: "_site", cache_control_for_etags: "no-cache"
+  plug Plug.Static, at: "/#{basepath}", from: "_site", cache_control_for_etags: "no-cache"
```

Comment:
I would also use `Path.join/2` here as well.

Reply:
I used `Path.join/2`, and also fixed the warning associated with `Application.get_env/2`.